### PR TITLE
Detail styling

### PR DESCRIPTION
**Pull request by karelhala**
selection_013

### CODE REVIEW

File: src/SmartComponents/Inventory/EntityDetail.js [hunk 1]
```diff
@@ -25,9 +36,90 @@ class EntityDetails extends Component {
...
+            </Grid>
+        );
+    }
+
+    generateFacts = () => {
+        return (
+            <Grid className="ins-entity-facts">
+                <GridItem md={ 6 }>
+                    <div>
+                        <span>
```
Comment (ryelo) [inline src/SmartComponents/Inventory/EntityDetail.js]: Should this be a `<Level>`?

Comment (ryelo) [inline src/SmartComponents/Inventory/EntityDetail.js]: Should this be a` <Level>`?

Comment (karelhala) [reply src/SmartComponents/Inventory/EntityDetail.js]: Not really Level will split these two into seperate items, but here we want to have them aligned next to each other.

**Pull request merged**